internal/cmd/gocdk/init.go

```diff
@@ -31,9 +31,11 @@ import (
 func init_(ctx context.Context, pctx *processContext, args []string) error {
 	f := newFlagSet(pctx, "init")
 	var modpath string
+	var allowExistingDir bool
 	f.StringVar(&modpath, "module-path", "", "the module import path for your "+
 		"project's go.mod file (required if project is outside of GOPATH)")
 	f.StringVar(&modpath, "m", "", "alias for --module-path")
+	f.BoolVar(&allowExistingDir, "allow-existing-dir", false, "true to allow initializing an existing directory (contents may be overwritten!)")
```

Comment:
Please add a TODO before this line referencing #1918.

Reply:
Done. I also updated the existing TODO down below to refer to it as well.